frontend/src/App.tsx

```diff
@@ -1,7 +1,33 @@
 import wailsLogo from './assets/wails.png'
 import './App.css'
+import * as Display from '../wailsjs/go/main/DisplayServer'
+import * as rt from '../wailsjs/runtime/runtime'
 
 function App() {
+
+    // Set Slide Data
+    var verse = 1
```

Comment:
This is just some dumb code added to the frontend to loop back and forth between a theoretical "song slide".

This is just to test the logic of sending data to the display server.